src/app.ts

```diff
@@ -0,0 +1,11 @@
+import "./config/env";
```

Comment:
I saw that you did this:
> Create src/config/env to configure where is located the .env file and use its enviroment variables. To use it, we just need to import this file (import './config/env')

What are the real advantages of this? why not take `.env` directly?

Reply:
That's because since we are using ES6+ syntax for ```import``` and ```export``` it's not possible configure automatically the ```Dotenv``` like ```require('dotenv').config()```. So to avoid the necessity of importing the ```config``` method and configuring where's the ```.env```, we just import ```./config/env```

Reply:
ok!